### PR TITLE
changed id name for drink class in index, changes to drinkscript

### PR DESCRIPTION
**Pull request by aksCo**

### CODE REVIEW

File: drinkscript.js [hunk 1]
```diff
@@ -1,29 +1,41 @@
 $(document).ready(function() {
+    var searchResults;
```
Comment (malakhim) [inline drinkscript.js]: isn't this the same global variable that the meals script uses? i feel like you'll run into conflicts if you both have a global variable named the same thing...

File: drinkscript.js [hunk 2]
```diff
@@ -1,29 +1,41 @@
...
-        searchDrinks(drinks);
+    function registerClickListeners() {
+        $(".drinkClass").on('click', grabInput);
     }
 
     function searchDrinks(drinks) {
         saveDrinksToLS(drinks);
         //Food Ajax Call
-        var queryURL = "https://www.thecocktaildb.com/api/json/v1/1/search.php?s=${drinks}"
+        var queryURL = `https://www.thecocktaildb.com/api/json/v1/1/search.php?s=${drinks}`
```
Comment (malakhim) [inline drinkscript.js:24]: hooray for backticks!

File: drinkscript.js (hunk 1 above)
Comment (aksCo) [reply drinkscript.js]: oh yea good point, I'll change it

File: drinkscript.js (hunk 2 above)
Comment (aksCo) [reply drinkscript.js:24]: heheh

**Review by MissNG-Git — APPROVED** [review]
Backtick implementation success!

**Pull request merged**